xchange-kraken/src/main/java/org/knowm/xchange/kraken/KrakenUtils.java

```diff
@@ -54,11 +54,18 @@ public static void setKrakenAssetPairs(Map<String, KrakenAssetPair> pairs) {
     }
   }
 
+  public static String getAltName(String key, KrakenAsset value) {
+    if(key.equals("XBT")) {
+      return "BTC";
+    }
```

Comment:
won't this break existing code that already assumes that XBT is the code for bitcoin on this exchange

please read this: https://github.com/knowm/XChange/issues/2394

until an universal solution is found we should assume that we operate on currency code as they are used on a given exchange